src/js/views/globals/modal/modal.hbs

```diff
@@ -11,7 +11,7 @@
 
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
-    <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button class="modal-cancel button--text js-close">{{ cancelText }}</button>
+    <button type="submit" class="button--blue {{ buttonClass }} js-submit">{{ submitText }}</button>
```

Comment:
Ah so I mean that the value of `{{ buttonClass }}` should default to `blue--button` and no class should be added here.

Reply:
Ah, that makes way more sense.